staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -1804,11 +1804,20 @@ type CSIPersistentVolumeSource struct {
 	// ControllerExpandSecretRef is a reference to the secret object containing
 	// sensitive information to pass to the CSI driver to complete the CSI
 	// ControllerExpandVolume call.
-	// This is an alpha field and requires enabling ExpandCSIVolumes feature gate.
+	// This is an beta field and requires enabling ExpandCSIVolumes feature gate.
 	// This field is optional, and may be empty if no secret is required. If the
 	// secret object contains more than one secret, all secrets are passed.
 	// +optional
 	ControllerExpandSecretRef *SecretReference `json:"controllerExpandSecretRef,omitempty" protobuf:"bytes,9,opt,name=controllerExpandSecretRef"`
+
+	// NodeExpandSecretRef is a reference to the secret object containing
```

Comment:
None of the field names in godoc in this struct are correct - they should all be `camelCase`, not `CamelCase`.  Can you open a follow up to make that change (not critical for this PR).

Reply:
sure, will track it with a follow up change. thanks!

Reply:
@zhucan Can you please address this comment?  Change "NodeExpandSecretRef" to "nodeExpandSecretRef".